crates/host/src/wasmbus/mod.rs

```diff
@@ -3727,6 +3780,30 @@ fn ensure_actor_capability(
     Ok(())
 }
 
+fn injector_to_headers(injector: &OtelHeaderInjector) -> HeaderMap {
```

Comment:
Also looks like you wanted to implement `From<&OtelHeaderInjector> for HeaderMap` in `tracing` crate (it's already bloated with dependencies, but if you really want, you could make `async_nats` an optional dep there)

Reply:
I think Taylor alluded to this in another comment, but I kept this out of `tracing` to avoid "polluting" that crate with the async-nats deps, to avoid transitive dependency issues down the line, when/if we want to update async-nats in one place (since `tracing` is used by both the host as well as providers)